source/includes/stackpath/_images.md

```diff
@@ -0,0 +1,106 @@
+## Images
+
+StackPath Edge Computing makes use of images to deploy virtual machines within a workload. The image can be from StackPath default images or
+a custom one. Only non-deprecated images are returned by default.
+
+Deploy and manage your Images.
+
+
+<!-------------------- LIST IMAGES -------------------->
+
+### List images
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/stackpath/test-area/images"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "stackId": "a8050b2b-39eb-4929-bce5-1af42055903e",
+      "id": "acad328a-153c-405b-a94d-d03479010e91",
+      "family": "centos7",
+      "tag": "v20201110",
+      "createdAt": "2020-11-10T20:33:32.609434Z",
+      "description": "Image using centos7",
+      "reference": "a8050b2b-39eb-4929-bce5-1af42055903e/centos7:v20201110",
+      "status": "READY"
+    },
+    {
+      "stackId": "a8050b2b-39eb-4929-bce5-1af42055903e",
+      "id": "c6f9f9c4-31a3-4114-946e-0bb82883301a",
+      "family": "ubuntu",
+      "tag": "v20201110",
+      "createdAt": "2020-11-10T20:34:11.328575Z",
+      "description": "Image using ubuntu",
+      "reference": "a8050b2b-39eb-4929-bce5-1af42055903e/ubuntu:v20201110",
+      "status": "READY"
+    }
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/images</code>
+
+Retrieve a list of all images in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | An image's unique identifier. It is the combination of /.
```

Comment:
Looks like we're missing something here 👀 (also in attribute table for get image)